src/Components/CommentList.jsx

```diff
@@ -3,19 +3,66 @@ import { useEffect } from "react";
 import { Grid } from "@mui/material";
 import CommentCard from "./CommentCard";
 import { getCommentsById } from "../api";
+import * as api from "../api";
 
 const CommentList = ({ review_id }) => {
   const [commentList, setCommentList] = useState([]);
+  const [commentUsername, setCommentUsername] = useState("");
+  const [commentBody, setCommentBody] = useState("");
+  const [successfulPost, setSuccessfulPost] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
 
   useEffect(() => {
     getCommentsById(review_id).then((comments) => {
       setCommentList(comments);
     });
   });
 
+  const commentToSubmit = {
+    username: commentUsername,
+    body: commentBody,
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+    api.postComment(commentToSubmit, review_id).then(() => {
+      setSuccessfulPost(true);
+      setIsLoading(false);
+    });
+  };
+
   return (
     <section>
       <h3>Comments:</h3>
+      <section type="CommentForm">
+        <h4>Post your own comment:</h4>
+        <form onSubmit={handleSubmit}>
+          <label htmlFor="username">State your username:</label>
+          <input
+            id="username"
+            value={commentUsername}
+            onChange={(event) => {
+              setCommentUsername(event.target.value);
+            }}
+            required
+          />
+          <br></br>
+          <label htmlFor="body">Comment:</label>
+          <input
```

Comment:
It would be more appropriate to use a textArea html tag here so the user can see the entire comment